### DISCUSSION TITLE
Royale-specific solo challenges

### DISCUSSION THREAD

**Discussion by altersaddle** [BattlesnakeOfficial/feedback#137] [General]
Here are some ideas for Royale mode challenges - designed for a single snake to play.  I'm not sure if a challenge royale mode could/should use a predictable "sauce spawn" rule or if it should match the actual gameplay which introduces it at random.  My preference is to stay true to the rules and keep it random.

**No Sauce Please**
Survive for a certain number of turns without your snake head ever being inside the sauce.  Variant: Survive the maximum number of turns before your snake runs out of health or runs into the hazard sauce.

**Just a Taste**
Survive for a certain number of turns while spending a certain amount of time (moves) with your snake head in the hazard sauce.  Alternative calculation/goal: Your snake uses more life points in the sauce than outside the sauce (assuming one move outside the sauce costs 1 health and one move inside costs 15, that means you must spend 1/15 of your time in the sauce, probably)

**I Was Born In It**
To earn this challenge reward, at least half of your snakes' moves are from within the hazard sauce at game end.  This challenge requires planning and luck, since you must survive until there is sufficient sauce to navigate for long stretches AND you must survive those disrespectful dives!

**Comment by bvanvugt** [comment]
Yes! We should do this. Overall we need a kinder way to introduce hazard sauce to new players.

**Comment by altersaddle** [comment]
More variants!

**Dive to Survive**
Uses a Large 19x19 board.  The snake is spawned at the center.  Some section (3-4 squares?) of the edge is hazard; hazards do not grow over time.  Food only spawns in the hazard zone.  Your snake must survive beyond 100 turns.

**Nowhere Left To Hide**
Regular royale rules.  Your snake must survive until the entire board is covered in hazards.  This one came up during a twitch stream but I can't remember who mentioned it.
